tunnel/wstuncli.go

```
@@ -395,6 +404,21 @@ func (wsc *WSConnection) writeStatus() {
 	fmt.Fprintf(wsc.tun.StatusFd, "Time: %s\n", time.Now().UTC().Format(time.RFC3339))
 }
 
+func (t *WSTunnelClient) wsDialerLocalPort(network string, addr string, ports []string) (conn net.Conn, err error) {
+	for _, port := range ports {
+		client, _ := net.ResolveTCPAddr("tcp", ":"+port)
```

Do not ignore errors.